src/main/java/com/bob/jr/TextToSpeech/TextToSpeech.java

```diff
@@ -14,12 +18,52 @@
 
 public class TextToSpeech {
 
-    private static final Map<Member, MemberVoiceConfig> memberVoiceMap = new HashMap<>();
+    private static final Map<Snowflake, MemberVoiceConfig> memberVoiceMap = new HashMap<>();
+    private static final String NEARLINE_BOBJR_SETTINGS = "nearline-bobjr-settings";
+    private static final String MEMBER_VOICE_SETTINGS_FOLDER = "member-voice-settings";
     private final TextToSpeechClient textToSpeechClient;
     private final Logger logger = LoggerFactory.getLogger(TextToSpeech.class);
 
+    private final Optional<Storage> gStorageOptional;
+
     public TextToSpeech() throws IOException {
+        this(null);
```

Comment:
This feels weird. Not saying it is wrong or anything just that it seems weird.

Reply:
We could make it an Optional<Storage>